### PR TITLE
Sitewide Links fix for IE11

### LINKED ISSUE #2316: Sitewide Links display incorrectly in IE11
**Describe the bug**
Sitewide links display vertically instead of horizontally in the grid in IE11. This is because we use `@supports not grid`, but this doesn't work in IE11. https://github.com/bbc/psammead/blob/latest/packages/components/psammead-sitewide-links/src/List/index.jsx#L84

**To Reproduce**
Open up any site or Psammead storybook in IE11 and look at the sitewide links in the footer.

**Expected behaviour**
They should behave like the other browsers and display in horizontal grid.

**Screenshots**
How it looks in IE11:

igbo footer
sitewidelinks

How it should look:
Screen Shot 2019-10-07 at 09 24 16
Screen Shot 2019-10-07 at 09 24 34

**Desktop (please complete the following information):**

- Browser: IE11

**Testing notes**
Need to check the component in all browsers we support to make sure it looks consistent.

- [x] This bug fix is expected to need manual testing.

**Additional context**

### PR DESCRIPTION
**Pull request by j-pendlebury**
Resolves https://github.com/bbc/psammead/issues/2316

**Overall change:** Add column support to IE11 without `grid`

**Code changes:**

- Remove `@supports not` because they don't work on IE11 👎 
- Use `column-count` to select how many columns there should be per breakpoint, same as on Chrome
- Use `column-span` for the first `li` to make sure the first row spans all columns
- Add `column-gap` for the spacing

## Screenshots:

Sitewide Links on Chrome [TOP is current LIVE component, BOTTOM is component with changes in this PR]

Screen Shot 2019-10-08 at 11 02 10

---
Sitewide Links on IE11 with fix

Screen Shot 2019-10-08 at 11 05 59

---
Sitewide Links on Chrome with `display: grid` disabled

image

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Automated jest tests added (for new features) or updated (for existing features)
- [x] This PR requires manual testing

### CODE REVIEW

File: packages/components/psammead-sitewide-links/src/List/index.jsx [hunk 1]
```diff
@@ -73,17 +73,13 @@ const StyledList = styled.ul`
     padding: ${GEL_SPACING} 0;
     margin-bottom: ${GEL_SPACING};
     grid-column: 1/-1;
-    @supports not (${grid}) {
-      width: 100%;
-    }
+    width: 100%;
```
Comment (DenisHdz) [inline packages/components/psammead-sitewide-links/src/List/index.jsx:82]: Wondering if we would also need this:
```
@supports (${grid}) {
    width: initial;
}
```

File: packages/components/psammead-sitewide-links/src/List/index.jsx [hunk 2]
```diff
@@ -73,17 +73,13 @@ const StyledList = styled.ul`
...
   }
 `;
 
 const StyledListItem = styled.li`
+  display: inline-block;
   min-width: 50%;
```
Comment (DenisHdz) [inline packages/components/psammead-sitewide-links/src/List/index.jsx:88]: Same here:
```
@supports (${grid}) {
    display: block;
    min-width: initial;
 }
```

Comment (DenisHdz) [reply packages/components/psammead-sitewide-links/src/List/index.jsx]: @OlgaLyubin is doing something similar in her PR https://github.com/bbc/psammead/blob/67db16d966d8122333c2cbb5dcebd278dc7f1995/packages/components/psammead-useful-links/src/index.jsx

File: packages/components/psammead-sitewide-links/src/List/index.jsx (hunk 1 above)
Comment (j-pendlebury) [reply packages/components/psammead-sitewide-links/src/List/index.jsx:82]: I could add this just to be safe but it doesn't look like the component is affected by this change if you look at the screenshots above

Comment (DenisHdz) [reply packages/components/psammead-sitewide-links/src/List/index.jsx:82]: Yep, just tested and it works fine without this, as when supporting grid the first list item will take the whole column 👍🏻

**Review by DenisHdz — APPROVED** [review]

**Comment by j-pendlebury** [comment]
image

Chromatic comparison. No changes in Chrome which supports CSS Grid

**Review by jroebu14 — APPROVED** [review]

**Comment by j-pendlebury** [comment]
image
PR went back to In Progress because the issue above was found during testing ^

File: packages/components/psammead-sitewide-links/src/List/index.jsx [hunk 3]
```diff
@@ -73,16 +73,16 @@ const StyledList = styled.ul`
...
-  @supports not (${grid}) {
-    display: inline-block;
+
+  @media all and (-ms-high-contrast: none), (-ms-high-contrast: active) {
```
Comment (DenisHdz) [inline packages/components/psammead-sitewide-links/src/List/index.jsx]: Is this to detect we are on IE? What about using `column-gap`, as we did in the `UsefulLinks`?
https://github.com/bbc/psammead/pull/2313/files#diff-72dd5167119d67cc68065f80ca0f141fR39

Comment (j-pendlebury) [reply packages/components/psammead-sitewide-links/src/List/index.jsx]: do you mean use `column-gap` inside of the `@media` block or instead of it?

Comment (DenisHdz) [reply packages/components/psammead-sitewide-links/src/List/index.jsx]: Instead of it and adding padding, sorry.

Comment (j-pendlebury) [reply packages/components/psammead-sitewide-links/src/List/index.jsx]: but then how do we make sure this is only added to IE? that was my issue with this change. The media query looks horrible but it was the only way (that I know of) to make sure the padding is only added to IE

Comment (j-pendlebury) [reply packages/components/psammead-sitewide-links/src/List/index.jsx]: Moved this discussion to a private chat and we used a solution similar to the Useful Links IE fix [here](https://github.com/bbc/psammead/blob/latest/packages/components/psammead-useful-links/src/index.jsx#L38-L39).

**Review by DenisHdz — APPROVED** [review]
LGTM, thanks for improving it 🚀

**Comment by jamesbrumpton** [comment]
lgtm

**Pull request merged**